<?xml version="1.0" encoding="UTF-8"?>
<ui version="4.0">
 <class>Kas</class>
 <widget class="QDialog" name="Kas">
  <property name="geometry">
   <rect>
    <x>0</x>
    <y>0</y>
    <width>944</width>
    <height>581</height>
   </rect>
  </property>
  <property name="windowTitle">
   <string>Catatan Kas Perpustakaan</string>
  </property>
  <layout class="QVBoxLayout" name="verticalLayout">
   <item>
    <layout class="QHBoxLayout" name="horizontalLayout">
     <item>
      <widget class="QLabel" name="label">
       <property name="text">
        <string>Rp.</string>
       </property>
      </widget>
     </item>
     <item>
      <widget class="QLineEdit" name="lUang">
       <property name="placeholderText">
        <string>Jumlah Uang</string>
       </property>
      </widget>
     </item>
     <item>
      <widget class="QLabel" name="label_2">
       <property name="text">
        <string>Ket. </string>
       </property>
      </widget>
     </item>
     <item>
      <widget class="QLineEdit" name="lKet">
       <property name="placeholderText">
        <string>Keterangan</string>
       </property>
      </widget>
     </item>
     <item>
      <widget class="QComboBox" name="comboJenis"/>
     </item>
     <item>
      <widget class="QPushButton" name="pSimpan">
       <property name="text">
        <string>Simpan</string>
       </property>
      </widget>
     </item>
    </layout>
   </item>
   <item>
    <widget class="QTableView" name="tblKas">
     <property name="font">
      <font>
       <pointsize>8</pointsize>
      </font>
     </property>
     <property name="alternatingRowColors">
      <bool>true</bool>
     </property>
    </widget>
   </item>
   <item>
    <layout class="QHBoxLayout" name="horizontalLayout_2">
     <property name="topMargin">
      <number>0</number>
     </property>
     <item>
      <widget class="QLabel" name="label_3">
       <property name="maximumSize">
        <size>
         <width>100</width>
         <height>16777215</height>
        </size>
       </property>
       <property name="text">
        <string>Pilih Bulan :</string>
       </property>
      </widget>
     </item>
     <item>
      <widget class="QComboBox" name="comboBulan"/>
     </item>
     <item>
      <widget class="QComboBox" name="comboTahun"/>
     </item>
     <item>
      <widget class="QPushButton" name="bRefresh">
       <property name="text">
        <string>Refresh</string>
       </property>
      </widget>
     </item>
     <item>
      <spacer name="horizontalSpacer">
       <property name="orientation">
        <enum>Qt::Horizontal</enum>
       </property>
       <property name="sizeHint" stdset="0">
        <size>
         <width>40</width>
         <height>20</height>
        </size>
       </property>
      </spacer>
     </item>
     <item>
      <widget class="QLabel" name="label_4">
       <property name="text">
        <string>Total Pemasukan :</string>
       </property>
      </widget>
     </item>
     <item>
      <widget class="QLabel" name="lTotalPemasukan">
       <property name="styleSheet">
        <string notr="true">font-weight: bold;</string>
       </property>
       <property name="text">
        <string/>
       </property>
      </widget>
     </item>
     <item>
      <widget class="QLabel" name="label_6">
       <property name="text">
        <string>Total Pengeluaran : </string>
       </property>
      </widget>
     </item>
     <item>
      <widget class="QLabel" name="lTotalKeluar">
       <property name="styleSheet">
        <string notr="true">font-weight: bold;</string>
       </property>
       <property name="text">
        <string/>
       </property>
      </widget>
     </item>
     <item>
      <widget class="QLabel" name="label_8">
       <property name="text">
        <string>Saldo Akhir :</string>
       </property>
      </widget>
     </item>
     <item>
      <widget class="QLabel" name="lSaldoAkhir">
       <property name="styleSheet">
        <string notr="true">font-weight: bold;</string>
       </property>
       <property name="text">
        <string/>
       </property>
      </widget>
     </item>
    </layout>
   </item>
  </layout>
 </widget>
 <resources/>
 <connections/>
</ui>
